docs/box2perm.html

```diff
@@ -0,0 +1,127 @@
+<html>
+<head>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+	<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>	
+	<title>box2perm utility</title>
+</head>
+<body>
+	<div class="container">
+		<h1>box2perm utility</h1>
+	
+		<p>Just paste your pre 2.5 INAV <strong>aux</strong> CLI lines and click "convert" to get a permanent ID aux lines</p>
+		
+		<div class="form-group">
+			<label for="inputText">Input aux CLI lines</label>
+			<textarea id="inputText" cols="40" rows="12" class="form-control"></textarea>
+		</div>
+
+		<input type="button" id="btnConvert" value="Convert" class="btn btn-primary">
+
+		<div class="form-group">
+			<label for="outputText">Output result</label>
+			<textarea id="outputText" cols="40" rows="12" class="form-control"></textarea>
+		</div>
+	
+	</div>
+	
+	<script>
+		var objBoxTable = [

```

Comment:
Nice stuff, but this is a yet another table we should maintain. Could this be somehow autogenerated?